metadata_scylla.go

```diff
@@ -0,0 +1,857 @@
+// +build !cassandra
+
+// Copyright (c) 2015 The gocql Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package gocql
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+)
+
+// schema metadata for a keyspace
+type KeyspaceMetadata struct {
```

Comment:
As far as I know, if something changes it's added, not deleted. For addition it's safe, since it pulls only what's currently available. Some fields are not strongly typed like TableMetadataOptions.Caching, Compaction, Compression, TableMetadata.Extensions or Flags.